app/Repositories/Project/ProjectRepository.php

```diff
@@ -120,4 +120,60 @@ public function fetchByLmsUrlAndLtiClient($lms_url, $lti_client_id)
         return $projects;
     }
 
+
+    /**
+     * To fetch project based on LMS settings
+     * @param Project $project
+     */
+    public function getProjectForPreview(Project $project)
+    {
+        $project = Project::where(['id'=> $project->id])
+            ->with('playlists.activities')
+            ->first();
+
+        $proj = [];
+        $proj["id"] = $project['id'];
+        $proj["created_at"] = $project['created_at'];
+        $proj["description"] = $project['description'];
+        $proj["name"] = $project['name'];
+        $proj["thumb_url"] = $project['thumb_url'];
+        $proj["updated_at"] = $project['updated_at'];
+        $proj["userid"] = $project['userid'];
```

Comment:
Please use camel case or snake_case. Also update this $project['userid'].

Reply:
done